src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -1842,6 +1841,56 @@ class CreateStatementSegment(BaseSegment):
     )
 
 
+@snowflake_dialect.segment(replace=True)
+class CreateViewStatementSegment(BaseSegment):
+    """A `CREATE VIEW` statement, specifically for Snowflake's dialect.
+
+    https://docs.snowflake.com/en/sql-reference/sql/create-view.html
```

Comment:
We're missing these options from the segment:
```
[ <col1> [ WITH ] MASKING POLICY <policy_name> [ USING ( <col1> , <cond_col1> , ... ) ]
           [ WITH ] TAG ( <tag_name> = '<tag_value>' [ , <tag_name> = '<tag_value>' , ... ] ) ]
  [ , <col2> [ ... ] ]
```
Any interest in including them in this PR? If not you should add a TODO note to the docstring to let people know it's incomplete.

Reply:
👍  Will add a note/TODO.

FYI - The complexity of column-level masking policy/tagging support is the `comma` that was added when multiple columns are involved, and per Snowflake's grammar, it doesn't specify that all column-level must be grouped together, making it difficult to craft the matching logic for it.

Reply:
Added a TODO: https://github.com/sqlfluff/sqlfluff/pull/2217/files#diff-a8da7f7dc8f0e448950f6d5c2cd6cd5590026ea6da32cdc8f233e037bea947ecR1885